app/src/main/java/es/wolfi/utils/ColorUtils.java

```diff
@@ -0,0 +1,50 @@
+/**
+ *  Passman Android App
+ *
+ * @copyright Copyright (c) 2017, Andy Scherzinger
+ * @license GNU AGPL version 3 or any later version
```

Comment:
reverted it, since according to the discussion on telegram, you can't give someone the copyright (except via a signed agreement).